lms/djangoapps/learner_dashboard/utils.py

```diff
@@ -13,3 +21,32 @@ def strip_course_id(path):
     """
     course_id = unicode(FAKE_COURSE_KEY)
     return path.split(course_id)[0]
+
+
+def disclaimer_incomplete_fields_notification(request):
+    """
+    Get the list of fields that are considered as additional but required.
+    If one of these fields are empty, then calculate the numbers of days
+    between the joined date and the current day to decide whether to display or not
+    the alert after a certain number of days passed from settings or site_configurations.
+    """
+    days_passed_threshold = configuration_helpers.get_value(
+        'DAYS_PASSED_TO_ALERT_PROFILE_INCOMPLETION',
+        settings.FEATURES.get('DAYS_PASSED_TO_ALERT_PROFILE_INCOMPLETION', 7)
+    )
```

Comment:
edX's [OEP communication re waffle flags](https://github.com/edx/open-edx-proposals/blob/e73537e2f3fadf2050cf266dfe15908435ab7b5c/oeps/oep-0017-feature-toggles.rst) that we started discussing on CAM-96 makes it clear that we should be using waffle switches for on/off feature gating instead of adding new `settings.FEATURES`, but they don't specify where to put other optional configuration settings. They do mention [integration with site configuration](https://github.com/edx/open-edx-proposals/blob/e73537e2f3fadf2050cf266dfe15908435ab7b5c/oeps/oep-0017-feature-toggles.rst#req-9-multi-tenancy), so I think the `settings.FEATURES` part can safely be removed here.